application/models/post_fixtures_test.go

```diff
@@ -86,12 +87,27 @@ func createFixturesForTestPost_manageStatusTransition_forwardProgression(ms *Mod
 	uf := createUserFixtures(ms.DB, 2)
 	users := uf.Users
 
-	posts := createPostFixtures(ms.DB, 2, 0, false)
+	posts := createPostFixtures(ms.DB, 4, 0, false)
 	posts[1].Status = PostStatusAccepted
 	posts[1].CreatedByID = users[1].ID
 	posts[1].ProviderID = nulls.NewInt(users[0].ID)
 	ms.NoError(ms.DB.Save(&posts[1]))
 
+	// Give the last Post an intermediate PostHistory
+	pHistory := PostHistory{
+		Status: PostStatusAccepted,
+		PostID: posts[3].ID,
+	}
+	mustCreate(ms.DB, &pHistory)
+
+	// Give these new statuses while by-passing the status transition validation
+	for i, status := range [2]PostStatus{PostStatusAccepted, PostStatusDelivered} {
+		id := i + 2
+
+		posts[id].Status = status
```

Comment:
Since actual ID is not the same as the `posts` index, using `id` may be misleading.

Reply:
:+1: